src/nvim/ex_cmds2.c

```diff
@@ -2374,7 +2374,7 @@ static FILE *fopen_noinh_readbin(char *filename)
   {
     int fdflags = fcntl(fd_tmp, F_GETFD);
     if (fdflags >= 0 && (fdflags & FD_CLOEXEC) == 0)
-      fcntl(fd_tmp, F_SETFD, fdflags | FD_CLOEXEC);
+      (void)fcntl(fd_tmp, F_SETFD, fdflags | FD_CLOEXEC);
```

Comment:
The function `fopen_noinh_readbin` is only called in ex_cmds2.c, and both `FD_CLOEXEC` and `HAVE_FD_CLOEXEC` are not used anywhere else. The whole code also works when this flag is not being set by `fcntl`, so we might as well ignore a return code that indicates failure.